doc/source/methods/PartialDependenceVariance.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<p>Backticks for <code>explain</code> arguments still required. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>